_rules/SC3-1-1-html-lang-valid.md

```diff
@@ -19,13 +19,13 @@ authors:
 
 ### Applicability
 
-The root element of the page, if it is an `html` element with a `lang` and/or `xml:lang` attribute.
+The root element of the page, if it is an `html` element with a [non-empty](#non-empty) `lang` and/or `xml:lang` attribute.
 
 ### Expectation
 
 The value of the `lang` or `xml:lang` attribute conforms to [BCP 47](https://www.ietf.org/rfc/bcp/bcp47.txt).
```

Comment:
Editorial: Consider removing the **Note** and writing the Expectation as:
The value of the `lang` attribute, or the `xml:lang` attribute if the `lang` attribute does not exist, conforms to [BCP 47](https://www.ietf.org/rfc/bcp/bcp47.txt).